### PR TITLE
pkg/process: support json trace output

### PR DESCRIPTION
**Pull request by jtolds**
What: support writing out json

Why: svgs are hard to parse!

Please describe the tests: no tests for this debug feature
 - Test 1:
 - Test 2:
 
Please describe the performance impact: none

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: pkg/process/exec_conf.go [hunk 1]
```diff
@@ -326,7 +326,11 @@ func cleanup(cmd *cobra.Command) {
 			if err != nil {
 				return err
 			}
-			err = present.SpansToSVG(fh, collect.CollectSpans(ctx, work))
+			if strings.HasSuffix(*traceOut, ".json") {
```
Comment (egonelbre) [inline pkg/process/exec_conf.go]: maybe switch on the `filepath.Ext`?

Comment (jtolds) [reply pkg/process/exec_conf.go]: i believe we have some folks already who are writing svgs out without an svg extension, so i didn't want to break those use cases
